superset/connectors/sqla/models.py

```diff
@@ -610,6 +612,20 @@ def query(self, query_obj):
             error_message = (
                 self.database.db_engine_spec.extract_error_message(e))
 
+        #fix for oracle db columns name matching issue - copied from cooleasyhan's fix for KeyError 953
```

Comment:
Instead of adding this kind of comments you could have picked the original commit instead